### PR TITLE
Implements installer generation for windows and debian based linux.

### PR DESCRIPTION
**Pull request by OmniBlade**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.10)
```
Comment (giulianobelinassi) [inline CMakeLists.txt:1]: Commit message should also say that it bumped cmake to version 3.14 from version 3.10.

**Comment by giulianobelinassi** [comment]
Debian package extract the contents into `/usr/local/bin`, which is binaries specifically created by the user. Instead, since this is a game, it should go on `/usr/games/`

**Comment by OmniBlade** [comment]
I wasn't actually familiar with the existence of /usr/games and looks like its use is somewhat varied between distros, but you are right, ubuntu seems to put games in /usr/games.

**Comment by OmniBlade** [comment]
PR broken due to unlinking with EA repo, see #716

**Pull request closed without merging**